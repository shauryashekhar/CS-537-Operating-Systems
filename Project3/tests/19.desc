Ignoring & for buit in commands
