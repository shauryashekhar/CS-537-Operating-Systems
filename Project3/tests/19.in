echo this is ok
wait 0 &
exit &
echo THIS SHOULD NOT PRINT