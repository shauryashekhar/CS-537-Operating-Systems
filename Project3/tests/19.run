./mysh tests/19.in
