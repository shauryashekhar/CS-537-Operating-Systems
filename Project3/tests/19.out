echo this is ok
this is ok
wait 0 &
exit &
